README.md

```diff
@@ -23,35 +23,50 @@ The jsep built files will be in the build/ directory.
 
 #### Client-side
 
-```javascript
-<script src="/PATH/TO/jsep.min.js"></script>
-...
-let parse_tree = jsep("1 + 1");
+```html
+<script type="module">
+import { Jsep } from '/PATH/TO/jsep.min.js';
+const parsed = Jsep.parse('1 + 1');
+</script>
 ```
 
 #### Node.JS
 
 First, run `npm install jsep`. Then, in your source file:
 
 ```javascript
-let jsep = require("jsep");
-let parse_tree = jsep("1 + 1");
+// ESM:
+import { Jsep } from 'jsep';
+const parse_tree = Jsep.parse('1 + 1');
+
+// or:
+import jsep from 'jsep';
+const parse_tree = jsep('1 + 1');
+
+
+// CJS:
+const { Jsep } = require('jsep');
+const parse_tree = Jsep.parse('1 + 1');
+
+// or: (NOTE: versions prior to 1.x didn't require `.default`)
+const jsep = require('jsep').default;
+const parsed = jsep('1 + 1');
 ```
 
 #### Custom Operators
 
 ```javascript
 // Add a custom ^ binary operator with precedence 10
-jsep.addBinaryOp("^", 10);
+Jsep.addBinaryOp("^", 10);
```

Comment:
Maybe we want to include import statements here to make it clear what these methods are called on?

Reply:
I've reverted these changes, and I think we can remove the identifier chars in a separate PR if we still want? It's newer, but isn't pretty minimal anyway. Depends how much of a breaking change the next release should be?